interpretation/src/main/kotlin/com/hartwig/actin/treatment/EvidenceScoringModel.kt

```diff
@@ -21,14 +21,25 @@ data class EvidenceScore(
 
 class EvidenceScoringModel(val config: ScoringConfig) {
 
-    fun score(treatment: TreatmentEvidence): EvidenceScore {
+    fun score(
+        treatment: TreatmentEvidence,
```

Comment:
perhaps the isIndirect should be a property of the MolecularMatchDetails in the TreatmentEvidence?